docs/FAQ/gui-faq-en.md

```diff
@@ -20,147 +20,155 @@ Zonemaster
+  - To protect the Engine from repetitive inputs, being the same IP
+    checking the same domain several times, there is a delay of 5 minutes between
+    subsequently identical tests.
+    This means that you can only test the same domain once every 5 minutes,
+    and if you try and test it again during that time the last results
+    will be displayed instead.
+  - Because Zonemaster was made to check domains, or DNS zones (e.g. 'zonemaster.net'), and
+    not hostnames in a domain (e.g. 'www.zonemaster.net'), Zonemaster will
+    report a failure if you try to test a hostname instead of a domain matching a
     DNS zone.
 
 <a name="q11"></a>
 #### 11. What kind of queries does Zonemaster generate?
-Zonemaster send multiple DNS queries to the name servers hosting the domain name and
-also to the name servers hosting the parent zone of the domain name.
+Zonemaster send multiple DNS queries to the name servers hosting the domain tested and
+also to the name servers hosting the parent zone of the domain tested.
 
-To get a full
-view of what queries and results are generated you can run the
-CLI interface (and in order to run the CLI interface you need to download the
-complete package and install it) and select full output.
-The output from the CLI tool is quite heavily technical
-so unless you are into bits and bytes you might want to skip this step.
+For the time being, the GUI interface of Zonemaster does not show any queries sent.
+Only the CLI interface can. If you want to see such queries, you will have to locally install
+a minimally working Zonemaster instance with both the Engine and CLI components.
+Queries sent can be shown using the 'DEBUG' level option.
+Fair warning, the output from the CLI can be quite heavy.
 
 <a name="q12"></a>
 #### 12. What is an undelegated domain test?
 An undelegated domain test is a test performed on a domain that may, or may not,
-be fully published in the DNS. This can be quite useful if one is going to move
-one's domain from one registrar to another,
-e.g., move zone example.com from the nameserver
-"ns.example.com" to the nameserver "ns.example.org". In this scenario one could perform
-an undelegated domain test providing the zone (example.com) and the nameserver you are moving to
-(ns.example.org) *before* you move your domain.
-When the results of the test are colour coded in green one can be fairly certain
-that the domain's new location is working well. However there
-might still be other problems in the zone data itself that this test is unaware of.
+be fully published in the DNS.
+This can be quite useful if one is going to move one's domain from one registrar to another,
+e.g., move zone 'example.com' from the name server 'ns.example.com' to the name server 'ns.example.org'.
+In this scenario one could perform an undelegated domain test providing the zone ('example.com')
+and the name server you are moving to ('ns.example.org') *before* you move your domain.
```

Comment:
While we’re at it: I think the word “migrate” is better than “move” in this paragraph.

Reply:
Updated.